lib/instrumentation/ids.js

```diff
@@ -0,0 +1,31 @@
+const { stringify } = require('querystring')
+
+class Ids {
+  toString () {
+    return stringify(this, ',', '=')
+  }
+}
+
+class SpanIds extends Ids {
+  constructor (span) {
+    super()
+    this['trace.id'] = span.traceId
+    this['span.id'] = span.id
```

Comment:
Wouldn't it be nicer if the user could access this without having to use quotes? E.g. `span.spanId` instead of `span['span.id']`

Reply:
I went with this form to align with the property names decided in the main design issue.

Reply:
That's for the output format when using the `toString()` version right? I was more concerned with those people who needed to access the values directly by their property name... or maybe I've misunderstood how they are supposed to be used/

Reply:
I understood it as _all_ forms of the logging object should look like that. @axw What do you think?

Reply:
@watson did you have a use case in mind? `apm.currentTraceIds.spanId` looks a bit nicer, but I'm not sure when you would want to do that. For log correlation purposes, I think it'll be typical to just log "the trace IDs", whatever they may be - span, transaction, or nothing, depending on the context and sampling.

Reply:
I was thinking that the user wanted to specify the keys in the structured logs them selfs. If so they needed to access these properties. But if they just want to use the keys we provide, then you're right, they don't need to access them explicitly.

Reply:
Talked about this in the weekly meeting. Let's go with the current implementation 👍